program/rust/src/tests/test_add_product.rs

```diff
@@ -57,13 +58,25 @@ fn test_add_product() {
     let mut product_setup_2 = AccountSetup::new::<ProductAccount>(&program_id);
     let product_account_2 = product_setup_2.as_account_info();
 
+    let mut permissions_setup = AccountSetup::new_permission(&program_id);
+    let permissions_account = permissions_setup.as_account_info();
+
+    {
+        let mut permissions_account_data =
+            PermissionAccount::initialize(&permissions_account, PC_VERSION).unwrap();
+        permissions_account_data.master_authority = *funding_account.key;
+        permissions_account_data.data_curation_authority = *funding_account.key;
+        permissions_account_data.security_authority = *funding_account.key;
+    }
```

Comment:
can you test that the old way of invoking these instructions fails now?

Reply:
err also would be good to test the specific authority that is allowed to do these operations. So maybe set that the other authorities are different

(unless this is already tested elsewhere, which i think it probably is)